From fb391a5050574d890d63942d76ecd472e8b9a567 Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Tue, 27 Sep 2011 16:54:46 +0200
Subject: Fix typos in man pages

Fix typos reported by lintian in man pages.

Origin: vendor
Forwarded: no
Last-Updated: 2020-08-10
---
 doc/vf_get_next_property.3      | 2 +-
 doc/vf_get_prop_value_time.3    | 2 +-
 doc/vf_parse_init.3             | 2 +-
 doc/vf_set_property_from_file.3 | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/doc/vf_get_next_property.3 b/doc/vf_get_next_property.3
index e6c491f..43e3460 100644
--- a/doc/vf_get_next_property.3
+++ b/doc/vf_get_next_property.3
@@ -14,7 +14,7 @@ extern bool_t vf_get_next_property(VF_PROP_T **pp_prop);
 .B "VF_PROP_T **pp_prop"
 Output pointer.
 .SH "DESCRIPTION"
-Find the next property given the current search critera.  User
+Find the next property given the current search criteria.  User
 calls vf_get_property() to locate first in a bunch of properties
 and can iterate over the rest by calling vf_get_next_property().
 .SH "RETURNS"
diff --git a/doc/vf_get_prop_value_time.3 b/doc/vf_get_prop_value_time.3
index cf446c9..3e6257c 100644
--- a/doc/vf_get_prop_value_time.3
+++ b/doc/vf_get_prop_value_time.3
@@ -32,7 +32,7 @@ Pointer to output time value.
 .SH "DESCRIPTION"
 Fetch a time_t value from a VF property.
 .SH "RETURNS"
-TRUE iff foudn & converted OK.
+TRUE iff found & converted OK.
 .SH "SEE ALSO"
 vf_parse_init(3),
 vf_parse_text(3),
diff --git a/doc/vf_parse_init.3 b/doc/vf_parse_init.3
index 3ffcc51..5a46f33 100644
--- a/doc/vf_parse_init.3
+++ b/doc/vf_parse_init.3
@@ -54,7 +54,7 @@ ret = FALSE;
 A parser allocated by vf_parse_init(), must be deallocated by calling
 vf_parse_end() whether or not parsing succeeds. Also, parsing may not
 be complete (ie. Values may be held buffered and not evaluated fully
-or assigned to a VF_OBJECT_T) untill the final call to vf_parse_end()
+or assigned to a VF_OBJECT_T) until the final call to vf_parse_end()
 => after a vf_parse_init() you _must_ vf_parse_end().
 .SH "RETURNS"
 TRUE iff parser allocated successfully.
diff --git a/doc/vf_set_property_from_file.3 b/doc/vf_set_property_from_file.3
index fc0d43f..276a110 100644
--- a/doc/vf_set_property_from_file.3
+++ b/doc/vf_set_property_from_file.3
@@ -32,7 +32,7 @@ Source filename.
 .SH "DESCRIPTION"
 Loads the indicated file into memory and sets the indicated property.
 .SH "RETURNS"
-TRUE iff succeded, FALSE else.
+TRUE iff succeeded, FALSE else.
 .SH "SEE ALSO"
 vf_parse_init(3),
 vf_parse_text(3),
